### ISSUE TITLE
Typo in Basic HTML and HTML5: Create a Set of Radio Buttons

### ISSUE THREAD

**Issue by asherdf** [freeCodeCamp/freeCodeCamp#17725]
There is a typo in the **3rd paragraph**, in the **2nd sentence**. The word "automatically" is missing the "m" to spell "autoatically".

https://learn.freecodecamp.org/responsive-web-design/basic-html-and-html5/create-a-set-of-radio-buttons/

* Browser Name: Firefox
* Browser Version: 52.8.1
* Operating System: Windows 10

**Comment by sbunny8** [comment]
Yes and the same typo appears on the next page about checkboxes; "automatically" is misspelled "autoatically".
https://learn.freecodecamp.org/responsive-web-design/basic-html-and-html5/create-a-set-of-checkboxes

**Comment by farhankarim** [comment]
Its my first contribution.
I am working on it.
